src/javascripts/Views/recordTreeView.js

```diff
@@ -52,12 +52,47 @@ ngapp.controller('recordTreeViewController', function($scope, $element, $timeout
         e.stopImmediatePropagation();
     };
 
-    $scope.onCellMouseDown = function(index) {
+    $scope.onCellMouseDown = function(e, node, index) {
+    
+        if (e.ctrlKey && index > 0 && node.value_type === xelib.vtReference)
+        {
+            index -= 1;
```

Comment:
index is only being used once in this code block, so you should just do `index - 1` on the next line